### ISSUE TITLE
Galleon's Graveyard - Invisible Wall Blocking Cannon Battle Room

### ISSUE THREAD

**Issue by forrester-marc** [Wynncraft/Issues#7307]
I've wasted three keys on this dungeon so far, the first time through I got trapped in the cutscene. (#7264,#7238)  The second and third time I killed the Armoured Pirate and opened the door to find an invisible wall blocking access to the cannon battle room, leaving me trapped in an area too small to use my teleport scroll.  Particularly frustrating as the Handcannon is the best bow for my build from 66 until max level.

**Comment by DebugMC** [comment]
Sadly there is no way to fix this.

**Comment by forrester-marc** [comment]
If that room is broken for the foreseeable future, perhaps the portal to the rest of the dungeon should be moved to the door behind the Armoured Pirate?

**Comment by DebugMC** [comment]
Not really, when talking to a builder about this before, I was told it was tried to fix the dungeon but it did not work.

**Comment by forrester-marc** [comment]
That's... an odd response from a programmer.  There are literally infinity possible ways to repair software problems.  O.o

**Comment by DebugMC** [comment]
Programer ≠ Builder

:P programmers would be the devs.

**Comment by forrester-marc** [comment]
In Minecraft that's not such a clear distinction, especially with all the command block sequences Wynncraft uses.  Still, even with no programming knowledge a builder could just copy the teleporter from the cannon battle room's exit to its entrance, bypassing the broken section.  Someone put days of work into building this dungeon, and in its present state no-one will experience any of it.

**Comment by ColinRioux** [comment]
Hi, all fixes to Galleon's dungeon will not be pushed to the main servers until the next update, which is soon (am not allowed to disclose the date here). I'm not fully sure if this one is fixed, but I can check tomorrow for your convenience. 

I also wouldn't group command blocking with programming, although it's similar in some sense, I think it's much different. In fact, command blocks manipulate commands, in which case they don't write code (aside from NBT data and sorts). Certainly there is a way to fix the issue like you said, but our process on build is very nit-picky, so please be patient with us. 

To recap– the bug you are experiencing won't be fixed asap. When the map on build is pushed to the main servers, which will most likely be when Corkus comes out, the bug should be handled and fixed. 

@DebugMC in the future, please don't say "there is no way to fix" instead either say it's being worked on or tag one of us, thank you.

**Comment by forrester-marc** [comment]
Yeah, they're distinct ideas, although with no very clear boundary in this case, MC being a turing complete programmable computer in its own right.  (https://www.youtube.com/user/sethbling)

Good to know something is working its way through the pipeline, obviously not everything can be first priority, but @DebugMC 's response was a little disconcerting.  No harm done though.  I guess I'll be using Decay Burner for a while.

**Comment by DebugMC** [comment]
@forrester-marc please, continue to tell me when asking the people who helped make the area and fix it when I was told they tried and when doing the other ways you described it was worse? How is that not trying.

**Comment by forrester-marc** [comment]
I wasn't suggesting that anyone wasn't trying, it just sounded like this dungeon had been abandoned indefinitely and might never be repaired.  That's not actually the situation, so everything's good.

**Comment by Razzgul** [comment]
I also ran into this problem but i think i found a way around it for the time being, when i was suspended i quit the game and then loaded up MC1.8.9 (as apposed to 1.11) and it seemed to work just fine

**Comment by Moe-Ronickah** [comment]
RESOLVED
